### ISSUE TITLE
[BUG] Covers + Thumbnails from other sources tend to be very low resolution

### ISSUE THREAD

**Issue by ConradWeiser** [Paperback-iOS/app#218]
**Describe the bug**
Covers and Thumbnails tend to be low quality. I'm pretty sure that a majority of sources are returning higher resolution covers. Is there some kind of downscaling being done in the app, which needs to be adjusted slightly?

**Comment by FaizanDurrani** [comment]
Yep, to keep the memory footprint low, the app downscales all the images. Otherwise on lower-end devices memory fills up resulting in a crash

**Comment by ConradWeiser** [comment]
I figured that'd be the case, although a lot of thumbnails look downright 100x300. - I'd like to have things scaled down __less__ if the device can handle it.

**Comment by FaizanDurrani** [comment]
The downscaling only happens on the library view mainly, everything else is full res

**Comment by ConradWeiser** [comment]
Not a bug - Intended functionality. Without downscaling as much, it poses a risk of crashing older iOS devices.
We're considering a setting which doesn't downscale images as much at risk of it using a lot more system memory (RAM)
It'll be something toggled in the settings, with an appropriate warning attached.

**Comment by celarye** [comment]
As part of a repository-wide cleanup for the 0.9 release, we are resetting our issue tracker to more efficiently track new feedback and feature requests. If this issue is still relevant or critical to your workflow, please feel free to reopen it with updated context. Thank you for your support!
